package-lock.json

```diff
@@ -267,7 +267,7 @@
 				"emotion-theming": "^10.0.27",
 				"interpolate-components": "^1.1.1",
 				"prop-types": "^15.7.2",
-				"react-stripe-elements": "^5.1.0"
```

Comment:
this one seems to be a leftover oddity from https://github.com/Automattic/wp-calypso/pull/39999 /cc @sirbrillig

Reply:
😬 ah yes, package-lock, my old nemesis. We meet again.